nomad/volumewatcher/interfaces.go

```diff
@@ -0,0 +1,28 @@
+package volumewatcher
```

Comment:
Note for review: these interfaces exist solely to give us handles for RPC mocks in the tests (so we don't have to set up clients with CSI plugins in unit tests).